tests/render/renderer/test_suite_scaffold_notebook_renderer.py

```diff
@@ -193,10 +82,22 @@ def test_notebook_execution_with_pandas_backend(titanic_data_context_no_data_doc
     context = DataContext(root_dir)
     obs_validation_result = context.get_validation_result(suite_name)
     assert obs_validation_result.statistics == {
-        "evaluated_expectations": 3,
-        "successful_expectations": 3,
+        "evaluated_expectations": 2,
+        "successful_expectations": 2,
         "unsuccessful_expectations": 0,
```

Comment:
Hi @talagluck this is a non-blocking question :  why is there 1 less expectation than before? which one is no longer included?